plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java

```diff
@@ -56,7 +54,10 @@
   public static final String CLIENT_APIS_GROUP = "clientapis"; //$NON-NLS-1$
 
   private static final Logger logger = Logger.getLogger(CloudLibraries.class.getName());
-  private static final ImmutableMap<String, Library> libraries = loadLibraryDefinitions();
+
+  // Mostly immutable info of libraries loaded from plugin.xml, but LibraryFile versions can be
+  // updated dynamically, e.g., to latest available release versions.
+  private static final Map<String, Library> libraries = loadLibraryDefinitions();
```

Comment:
Why did you remove the ImmutableMap type? That seemed useful.
However we probably shouldn't comment it as "Moslty immutable".

Reply:
OK, I'll revert it.